src/Symfony/Component/DependencyInjection/Tests/Fixtures/config/inline_static_constructor.expected.yml

```diff
@@ -0,0 +1,10 @@
+
+services:
+    service_container:
+        class: Symfony\Component\DependencyInjection\ContainerInterface
+        public: true
+        synthetic: true
+    foo:
+        class: Symfony\Component\DependencyInjection\Tests\Fixtures\Prototype\StaticConstructor\PrototypeStaticConstructorAsArgument
+        public: true
+        arguments: [!service { class: Symfony\Component\DependencyInjection\Tests\Fixtures\Prototype\StaticConstructor\PrototypeStaticConstructor, factory: [Symfony\Component\DependencyInjection\Tests\Fixtures\Prototype\StaticConstructor\PrototypeStaticConstructor, create] }]
```

Comment:
this makes me wonder if we couldn't add a rule to dump a constructor here (see YamlDumper, same in XmlDumper)

Reply:
That's updated! 👌